examples/research_projects/jax-projects/wav2vec2/run_wav2vec2_pretrain_flax.py

```diff
@@ -329,14 +328,20 @@ def main():
         model_args.model_name_or_path, cache_dir=model_args.cache_dir, do_normalize=True
     )
 
+    # check that all files have the correct sampling rate
+    datasets = datasets.cast_column(data_args.audio_file_column, Audio(sampling_rate=feature_extractor.sampling_rate))
+
     def prepare_dataset(batch):
-        # check that all files have the correct sampling rate
-        batch["speech"], _ = librosa.load(batch[data_args.speech_file_column], sr=feature_extractor.sampling_rate)
+        batch["speech"] = batch[data_args.audio_file_column]["array"]
```

Comment:
Fine since the latest version of datasets uses `librosa` to load audio files - can you make clear on the README that users should use the latest version of Datasets (v 2.14.5 or higher)? https://github.com/huggingface/transformers/tree/main/examples/research_projects/jax-projects/wav2vec2#wav2vec2-contrastive-loss-pretraining-examples

Reply:
I have documented in readme for lastest commit